shopify_api_extensions.gemspec

```diff
@@ -21,9 +21,9 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency 'shopify_api', '~> 4.7'
+  spec.add_dependency 'shopify_api', '~> 5.2.4'
```

Comment:
@w1zeman1p does this change only work with this new API version? If not, I'd rather not lock to a newer (and more specific) API version.

Reply:
I don't actually know, I have the newer version and it works for me.

Reply:
removed the gemspec changes